efficientdet/hparams_config.py

```diff
@@ -200,6 +200,8 @@ def default_detection_configs():
 
   h.backbone_name = 'efficientnet-b1'
   h.backbone_config = None
+  # Availabel values are 'fpn', 'backbone', None
+  h.freeze = None
```

Comment:
How about provide something like  "h.var_freeze_expr" (we can provide some examples in comments and tests)?

Then you can just define one function like:

def variable_filter_fn(variables, var_freeze_expr=None):
  matcher = re.compile(var_freeze_expr) if var_freeze_expr else None
  var_list = [v for v in variables if matcher and not matcher.match(v.name)]
  return var_list